ImportExport/CadbaseLibrary/README.md

```diff
@@ -48,10 +48,14 @@ In the **CADBase library** window, in the **Options** tab, click the **Settings*
 
 In FreeCAD, you can find which is your user **modules folder** by entering or pasting `App.getUserAppDataDir()+"Mod"` and your usr **macros folder** by entering `App.getUserMacroDir()` in the Python console (found under menu View->Panels)
 
+Please don't use `cadbase_response_file_2018` as the name of files or folders in the CADBase library folder.
```

Comment:
Why?

Reply:
Server responses are stored in this file, if the user saves his data in this file, he may lose them.